test/integration/test_any_holidays_during_work_week.rb

```diff
@@ -0,0 +1,90 @@
+require File.expand_path(File.dirname(__FILE__)) + '/../test_helper'
```

Comment:
This is a good solution.:)
Dividing test cases is easy for programmers to understand.